Mantle/MTLModel+NSCoding.m

```diff
@@ -222,7 +230,9 @@ - (void)encodeWithCoder:(NSCoder *)coder {
 	[coder encodeObject:@(self.class.modelVersion) forKey:MTLModelVersionKey];
 
 	NSDictionary *encodingBehaviors = self.class.encodingBehaviorsByPropertyKey;
-	[self.dictionaryValue enumerateKeysAndObjectsUsingBlock:^(NSString *key, id value, BOOL *stop) {
+	NSDictionary<NSString *, id> *dictionaryValue = [self dictionaryValue];
+
+	[dictionaryValue enumerateKeysAndObjectsUsingBlock:^(NSString *key, id value, BOOL *stop) {
```

Comment:
Can this change be dumped?

Reply:
I'm not sure what you mean - The alternative would be to either cast the block, or cast the `self.dictionaryValue`, otherwise I get 
```
Incompatible block pointer types sending 'void (^)(NSString *__strong, __strong id, BOOL *)' to parameter of type 'void (^)(__strong id<NSCopying>, __strong id, BOOL *)'clang(typecheck_convert_incompatible_block_pointer)
```
That's the purpose of putting it in the `NSDictionary<NSString *, id>`

Reply:
Oh, I see. It wasn't clear that was avoiding the cast. Never mind :)